governance/polls/Add GUNI-DAI-USDC as a new Vault Type - September 6, 2021.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Add GUNI-DAI-USDC as a new Vault Type - September 6, 2021
+summary: This poll proposes system parameters which could be used to initialize GUNI-DAI-USDC as a new vault type.
+discussion_link: https://forum.makerdao.com/t/g-uni-dai-usdc-uniswap-v3-lp-erc20-collateral-onboarding-application/9026
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
```

Comment:
We need a start-date and end-date field. Like, make sure that you are using the most up to date version of the template.

Reply:
It's not the first time this happens. I open up a template and there's no "start-date, end-date", I usually have to manually add it. Don't know why. I'll keep an eye on it.